test/integrations/component.test.ts

```diff
@@ -111,6 +177,13 @@ const sourceTestHandler = async (tcData) => {
   )}`;
   await testRoute(route, tcData);
 };
+// all the axios requests will be stored in this map
+const allAxiosReqsMap = allTestDataFilePaths.reduce((agg, currPath) => {
```

Comment:
should we build this per test file and reset again? example: when we running pardot then it it has only maps build for it and reset after it is completed, is it easy to do?

Reply:
Don't think it is easily possible

Reply:
then it is fine.